fittotcx/program.py

```diff
@@ -234,12 +234,17 @@ def main():
             description="This program takes a FIT file and converts it " +
                         "into an TCX file and output the result to the " +
                         "standard output.")
-    parser.add_argument('file', metavar='FILE', type=argparse.FileType('r'))
+    parser.add_argument('inputfile', metavar='INPUTFILE', nargs=1, type=str)
```

Comment:
Should stay as `type=argparse.FileType('r')`, with `default=sys.stdin`.

`fitparse`'s `FitFile` class can already handle a file object in place of a filename. https://github.com/dtcooper/python-fitparse/blob/master/fitparse/utils.py#L38